### DISCUSSION TITLE
UseForm defaultValues

### DISCUSSION THREAD

**Discussion by NiroshKumarR** [react-hook-form/react-hook-form#10971] [Q&A]
is this issue resolved,https://github.com/react-hook-form/react-hook-form/issues/3638#issue-760164237

I'm facing the same scenario, can any one help me with this.

**Comment by Moshyfawn** [comment]
#3638 has been closed with a resolution from Bill 3 years ago 😉

With v7, you might want to look into the [`values`](https://react-hook-form.com/docs/useform#values) API for convenience.
